lib/core/src/Cardano/DB/Sqlite.hs

```diff
@@ -206,35 +207,63 @@ destroyDBLayer (SqliteContext {getSqlBackend, trace, dbFile}) = do
                            Internal / Database Setup
 -------------------------------------------------------------------------------}
 
+-- | Represents an automatic migration to be performed on a database.
+data AutoMigration
+    = AutoMigrationNone
+      -- ^ Do not perform an automatic migration.
+    | AutoMigrationOnCreate Migration
+      -- ^ Perform the specified automatic migration when creating a new
+      -- database.
+    | AutoMigrationOnCreateOrRestart Migration
+      -- ^ Perform the specified automatic migration when creating a new
+      -- database or when restarting a pre-existing database.
```

Comment:
What about simply calling this `AutoMigrationAlways` ?

Reply:
@KtorZ wrote:

> What about simply calling this `AutoMigrationAlways` ?

Fair enough! I don't have a strong opinion about this. I've changed it to match your suggestion.

**Fixed in:** https://github.com/input-output-hk/cardano-wallet/pull/1334/commits/50978cc5084eb6c892e617ef2fc890925a136fa4